.github/CODEOWNERS

```diff
@@ -154,7 +154,7 @@
 
 # PRLabel: %Cognitive - Language
 # ServiceLabel: %Cognitive - Language %Service Attention
-/sdk/cognitivelanguage/                                            @heaths @rokulka @mshaban-msft
+/sdk/cognitivelanguage/                                            @quentinRobinson @wangyuantao @heaths @rokulka @mshaban-msft
```

Comment:
`mshaban-msft` needs to join the Microsoft and Azure organizations before they can be added.

Reply:
`wangyuantao` has already joined the Microsoft and Azure organizations, but I see that he's missing write access. He's currently OOF, but I'll ping him about it again when he's back.

Interestingly, `mshaban-msft` was already there. @heaths: Do you think we should keep these codeowners here? Or would you recommend replacing them with the new ones?

Reply:
That's my bad.  I misread `mshaban-msft` as also being added.   Someone dropped the ball on that, but we shouldn't pull an existing alias, assuming they are still working on the product.  If you know them and could gently nudge them to get their memberships updated, that would be much appreciated.   

There's no requirement for write access - but it would definitely be helpful if they're going to be working in the repo.

Reply:
I have confirmed that we can remove `rokulka` and `mshaban-msft`. The former does not work on this product anymore, and the latter has left Microsoft. I have updated the PR.